src/Proj.php

```diff
@@ -374,7 +375,7 @@ public function loadProjCode($projName)
         }
 
         // The class name for the projection code
-        $classname = '\\proj4php\\projCode\\' . ucfirst($projName);
+        $classname = '\\proj4php\\projCode\\' . ucfirst($projName ?? "");
```

Comment:
This was annother deprecation warning that occurred. Did not expect to encounter or fix this.